src/app/(homepage)/Demo.js

```diff
@@ -2,30 +2,38 @@ import OosStepper from "@/components/OosStepper";
 import { Box, Container } from "@mui/material";
 import Typography from "@mui/material/Typography";
 
-function DemoHeader({sx}) {
+
+function DemoHeader({ sx, subheading1, heading1, heading2}) {
   return (
-    <Box sx={{textAlign: "center", maxWidth: 735, ...sx}}>
-      <Typography variant="subheading">
-        Directly from the calendar
-      </Typography>
-      <Typography mt={2} variant="h2">
-        Discover the ease of booking activities without the text and email tennis
-      </Typography>
+    <Box sx={{ textAlign: "center", maxWidth: 735, ...sx }}>
+      {subheading1 && <Typography variant="subheading">{subheading1}</Typography>}
+      {heading1 && <Typography mt={2} variant="h2">{heading1}</Typography>}
     </Box>
-  )
+  );
 }
 
-export default function Demo() {
-  const steps = [
-    "Find the activities your child will love",
-    "Add them to child’s calendar",
-    "Review and then pay for all in one click",
-  ];
+function Demo(props) {
   return (
     <Container sx={{ py: { xs: 6, md: 10 } }}>
-      <DemoHeader sx={{ mx: "auto" }} />
-      <OosStepper sx={{ mt: 4, maxWidth: 700, mx: "auto" }} steps={steps} />
-      <Box component="img" src="/demo.png" sx={{ mt: { xs: 2, sm: 10 }, width: "100%" }} />
+      <DemoHeader sx={{ mx: "auto" }} subheading1={props.subheading} heading1={props.heading1}/>
+      <OosStepper sx={{ mt: 4, maxWidth: 700, mx: "auto" }} steps={props.steps} />
+      <Box component="img" src={props.imageUrl} sx={{ mt: { xs: 2, sm: 10 }, width: "100%" }} />
     </Container>
   );
-}
\ No newline at end of file
+}
+
+const HomepageDemo = () => (
```

Comment:
No need for a separate component. It does nothing except of adding 1 extra place to check.
It is more readable to inline (replace this component invokation with actual definition where it is called).